spec/index.html

```diff
@@ -495,25 +505,42 @@ <h3>Blank nodes</h3>
       without using an IRI to <a>identify</a> any particular thing.
       This is not the same as assuming that the blank node indicates an 'unknown' IRI.</p>
 
-    <p>Suppose I is a simple interpretation and A is a mapping from a set of blank nodes
-      to the universe IR of I.
-      Define the mapping [I+A] to be I on <a>names</a>,
-      and A on blank nodes on the set: [I+A](x)=I(x) when x is a <a>name</a>
-      and [I+A](x)=A(x) when x is a blank node;
-      and extend this mapping to triples and RDF graphs using the rules given above for ground graphs.
-      Then the semantic conditions for an RDF graph are:</p>
+    <p>Suppose that</p>
+    <ul>
+      <li>I is a simple interpretation,</li>
+      <li>A is a mapping from a set N of blank nodes to the universe IR of I, and</li>
+      <li class="new">B is a mapping from a subset of N to the set of <a>ground</a> terms, such that for every x in that subset, I(B(x)) = A(x).</li>
+      <li>B is extended to names and quoted triples:
+        <ul>
+          <li>if x is a <a>name</a>, B(x) = x</li>
+          <li>if x is a quoted triple (s, p, o), B(x) = (B(s), B(p), B(o))</li>
+        </ul>
+      </li>
+    </ul>
+
+    <p>Define the mapping [I+A+B] as follows:</p>
+    <ul>
+      <li>if x is a <a>ground</a> term, [I+A+B](x)=I(x)</li>
+      <li>if x is a blank node, [I+A+B](x)=A(x)</li>
+      <li class="new">if x is a non-<a>ground</a> quoted triple, then [I+A+B](x)=I(B(x)) </li>
+      <li>and extend this mapping to triples and RDF graphs using the rules given above for ground graphs.</li>
+    </ul>
+
+    <p>Then the semantic conditions for an RDF graph are:</p>
 
     <table>
       <caption>Semantic condition for blank nodes.</caption>
       <tbody>
         <tr>
-          <td class="semantictable">If E is an RDF graph then I(E) = true if [I+A](E) =
-          true for some mapping A from the set of blank nodes in E to IR, otherwise
+          <td class="semantictable">If E is an RDF graph then I(E) = true if [I+A+B](E) =
+          true for some mappings A <span class="new">and B as defined above</span>, otherwise
```

Comment:
```suggestion
          <td class="semantictable">If E is an RDF graph, then I(E) = true if [I+A+B](E) =
          true for some mappings A <span class="new">and B as defined above</span>; otherwise,
```